build.sh

```diff
@@ -82,7 +93,8 @@ pushd ${FRAMEWORK_DIR} > /dev/null
 
 configure_ut_control() {
     cd ./ut-control
-    git checkout ${UT_CONTROL_PROJECT_VERSION}
+    #git checkout ${UT_CONTROL_PROJECT_VERSION}
+    git checkout feature/gh44-ut-control-cpp-support
```

Comment:
Make sure, this is reverted before merge

Reply:
you should do this now, and put it back in when you use it, you can commit it out and leave in if you like.

Reply:
#107 created.